scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -295,13 +299,15 @@ try {
     }
     Install-AdminRouterAgent
     $mesosFlags = Get-MesosFlags
+    $IsMatricsServiceInstalled = $false
     if(!$mesosFlags.authenticate_agents) {
         # Install Metrics only if mesos_authentication is disabled
         Install-MetricsAgent
+        $IsMatricsServiceInstalled = $true
     }
     # To get collect a complete list of services for node health monitoring,
     # the Diagnostics needs always to be the last one to install
-    Install-DiagnosticsAgent
+    Install-DiagnosticsAgent -IncludeMetricsToSericeList $IsMatricsServiceInstalled
```

Comment:
Ditto

Reply:
I will make sure this was not included once the other PR was merged.